airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -42,6 +42,7 @@
 from airflow.exceptions import AirflowException
 from airflow.providers.amazon.aws.exceptions import S3HookUriParseFailure
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+from airflow.providers.amazon.aws.utils.aws_api import format_tags
```

Comment:
Why the` aws_` prefix? Lets please double check with @ferruzzi as he lead the effort of making AWS provider comply with AIP-21

Reply:
I'm not sure what else you;'d name the module here that would make more sense?  It cold be boto_api or boto_utils maybe but I'm not sure either of those are any better, really?

Reply:
I named it boto_ something in my first commit, but then as Taragolis mentioned:

> this is not boto3 behaviour this how what AWS API expected this parameters

I changed it to aws

Reply:
Maybe a stupid idea but what if we place it in `airflow/providers/amazon/aws/utils/__init__.py` or it for generic stuff only?

Reply:
Pretty sure Jarek had mentioned not liking stuff in the __init__ files, so I've been trying to avoid it when I can think of a better name.  Sometimes it's easier than others.  Naming is hard.

In this case I think aws_api.py is a sound option.  @eladkal - do you definitely want it changed or were you just making sure I was good with it?

Reply:
`airflow/providers/amazon/aws/utils/tags.py`?

Reply:
I could get behind that.